src/domain/usecases/ValidateGlassUploadsUseCase.ts

```diff
@@ -26,7 +26,7 @@ export class ValidateGlassUploadsUseCase implements UseCase {
         return this.glassUploadsRepository
             .getAll()
             .flatMap(data =>
-                data.length === 0 ? this.glassUploadsRepository.save(submission) : Future.success(undefined)
+                data.length === 0 ? this.glassUploadsRepository.save(upload) : Future.success(undefined)
```

Comment:
Not familiar with the app, but isn't a bit weird that a use case doing `glassUploadsRepository.save(upload)` is called `ValidateGlassUploadsUseCase`?

Reply:
Noted in glass team slack, will be addressed in upcoming tasks.